miden-tx/tests/p2id_script_test.rs

```diff
@@ -235,13 +236,83 @@ fn test_p2id_script_multiple_assets() {
         .collect::<Vec<_>>();
 
     // Execute the transaction and get the witness
-    let transaction_result_2 = executor_2.execute_transaction(
+    let executed_transaction_2 = executor_2.execute_transaction(
         malicious_account_id,
         block_ref,
         &note_origins,
         Some(tx_script_malicious),
     );
 
     // Check that we got the expected result - TransactionExecutorError
-    assert!(transaction_result_2.is_err());
+    assert!(executed_transaction_2.is_err());
+}
+
+#[test]
+fn test_p2id_prove_and_verify() {
```

Comment:
Is there a benefit to adding a separate test like this? In `P2IDR` test, we just integrated proving/verification into the existing test. Should we do the same here?

Reply:
agreed.